aips/aip-81.md

```diff
@@ -173,20 +153,20 @@ The pepper request should be rejected if any check below fails.
 
 ### Pepper processing
 - Let `uid_val` be the user ID from `jwt.payload` indicated by `uid_key`.
-- Let `aud` be the effective client ID determined by `jwt.payload.aud` and `aud_override`.
-- Let `input` be a canonical serialization of data items `jwt.payload.iss, uid_key, uid_val, aud`.
-- Compute `pepper` as `vuf.eval(vuf_sk, input)`.
-- Let `pepper_bytes` be a serialization of the VUF output `pepper`.
-- Compute `pepper_encryped` as `asymmetric.enc(epk, pepper_bytes)`.
-
-### Pepper response (the output)
-
-The pepper response from the pepper service to the user should simply be the pepper encrypted by the EPK, hexlified, and wrapped in a JSON.
-
-Below is an example pepper response, assuming `vuf=Bls12381G1Bls` and `asymmetric=ElGamalCurve25519Aes256Gcm`.
+- Compute `pepper_input` to be a canonical serialization of data items `(jwt.payload.iss, uid_key, uid_val, jwt.payload.aud)`.
+- Compute `pepper_base` as `vuf.eval(vuf_sk, input)`.
+- Compute `master_pepper` as `H(pepper_base)`.
+- Optionally:
+  - compute `derived_pepper` from `master_pepper` and `derivation_path`,
+    following key derivation scheme [SLIP-0010](https://github.com/satoshilabs/slips/blob/master/slip-0010.md).
+  - compute the `IdCommitment` as defined in [AIP-61](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-61.md#public-keys) and derive the initial account address.
```

Comment:
Reminding the reader how `address` is derived here would be helpful too.

Reply:
fixed